workflow/main.rb

```diff
@@ -22,6 +22,15 @@ def emoji_item(emoji)
   }
 end
 
+def is_image_only(emoji)
+  if emoji.image_filename == "#{emoji.name}.png"
```

Comment:
The problem I'm having with just checking for `end_with` is that they *all* end with `.png`. gemoji separates valid unicode emoji into their own folder, but `image_filename` still returns the image, not the actual emoji so checks for this always return true and hide the results.

Since gemoji has the image-only ones in the main emoji directory, I'm able to check if the file exists in there and assume it's not valid unicode without having to dive down into the unicode directory. If there's another way to check, I don't know it.